src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import * as api from "../api";
+
+export default function ArticleVotes({ articleCard }) {
+  const [upvoted, setUpvoted] = useState(false);
+  const [resetUpvote, setResetUpvote] = useState(null);
+  const [downvoted, setDownvoted] = useState(false);
+  const [resetDownvote, setResetDownvote] = useState(false);
+  const [votes, setVotes] = useState(0);
+  const [error, setError] = useState(null);
+
+  //UPVOTE BUTTON
+  function increaseUpvote(event) {
+    setError(null);
+    event.preventDefault();
+    setVotes((currentVotes) => {
+      return currentVotes + 1;
+    });
+    api.updateArticleVotes(articleCard.article_id, 1).catch((error) => {
+      setError(error);
+    });
+    setUpvoted(true);
+    setResetUpvote(true);
+  }
+
+  function decreaseUpvote(event) {
+    setError(null);
+    event.preventDefault();
```

Comment:
No need to prevent this default behaviour, only need to do this for forms to prevent them navigating away from the page.